packages/typespec-test/test/healthInsights_radiologyinsights/generated/typespec-ts/vitest.browser.config.ts

```diff
@@ -1,38 +1,16 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { defineConfig } from "vitest/config";
-import { relativeRecordingsPath } from "@azure-tools/test-recorder";
+import { defineConfig, mergeConfig } from "vitest/config";
+import * as viteConfig from "../../../vitest.browser.shared.config.ts";
```

Comment:
this is incorrect to generate code in non-azure sdk repo because there is no relevant shared file there.

Reply:
let's disable the test relevant generation for unbranded for now and create an issue to track this effort.